v3/testdata/smime/subscriber_with_mixed_crl_distribution_points.pem

```diff
@@ -0,0 +1,47 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 3 (0x3)
+        Signature Algorithm: ecdsa-with-SHA256
+        Issuer: 
+        Validity
+            Not Before: Feb 13 19:20:31 2024 GMT
+            Not After : Nov 30 00:00:00 9998 GMT
+        Subject: 
+        Subject Public Key Info:
+            Public Key Algorithm: id-ecPublicKey
+                Public-Key: (256 bit)
+                pub:
+                    04:7d:26:16:25:1e:57:16:05:f0:e2:95:77:56:b8:
+                    f6:66:c0:ba:1a:35:fd:6c:57:3d:07:16:c7:fc:44:
+                    67:32:41:b9:f6:1e:94:91:ad:37:90:28:34:45:70:
+                    32:c0:9e:64:e8:9a:14:55:41:ff:19:87:fb:43:0b:
+                    25:c2:8d:3d:f5
+                ASN1 OID: prime256v1
+                NIST CURVE: P-256
+        X509v3 extensions:
+            X509v3 Extended Key Usage: 
+                E-mail Protection
+            X509v3 Certificate Policies: 
+                Policy: 2.23.140.1.5.1.3
+            X509v3 CRL Distribution Points: 
+                Full Name:
+                  URI:ldap://example.com
+                Full Name:
+                  URI:http://example.com
+    Signature Algorithm: ecdsa-with-SHA256
+    Signature Value:
+        30:45:02:21:00:eb:f3:1b:63:15:c8:38:41:05:24:c1:29:51:
+        12:23:99:d6:aa:86:a2:5e:37:eb:48:13:8b:51:19:33:97:f4:
+        c1:02:20:7b:02:fd:c7:4f:7d:ff:fd:1b:b2:7e:66:f8:b9:d5:
+        e5:be:8b:18:8a:f2:3e:55:33:84:dd:cb:ae:19:ad:c8:c0
+-----BEGIN CERTIFICATE-----
+MIIBXTCCAQOgAwIBAgIBAzAKBggqhkjOPQQDAjAAMCAXDTI0MDIxMzE5MjAzMVoY
+Dzk5OTgxMTMwMDAwMDAwWjAAMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEfSYW
+JR5XFgXw4pV3Vrj2ZsC6GjX9bFc9BxbH/ERnMkG59h6Uka03kCg0RXAywJ5k6JoU
+VUH/GYf7Qwslwo099aNsMGowEwYDVR0lBAwwCgYIKwYBBQUHAwQwFAYDVR0gBA0w
+CzAJBgdngQwBBQEDMD0GA1UdHwQ2MDQwGKAWoBSGEmxkYXA6Ly9leGFtcGxlLmNv
+bTAYoBagFIYSaHR0cDovL2V4YW1wbGUuY29tMAoGCCqGSM49BAMCA0gAMEUCIQDr
+8xtjFcg4QQUkwSlREiOZ1qqGol4360gTi1EZM5f0wQIgewL9x099//0bsn5m+LnV
+5b6LGIryPlUzhN3LrhmtyMA=
+-----END CERTIFICATE-----
```

Comment:
nit: resolve the marker?

Reply:
Added a newline (I think that's what you're referring to).